vue-handler.js

```diff
@@ -75,9 +75,9 @@ const componentList = {
             <thead><tr>
                 <td class="name"><i class="fad fa-book"></i> නම</td>
                 <td class="share"></td>
-                <td v-show="columns.includes('folder')" class="folder"><i class="fad fa-folder"></i> ගොනුව</td>
-                <td v-show="columns.includes('downloads')" class="downloads"><i class="fad fa-tally"></i><span class="ss-hide"> භාගත ගණන</span></td>
-                <td v-show="columns.includes('date_added')" class="date_added"><i class="fad fa-calendar"></i><span class="ss-hide"> එකතුකළ දිනය</span></td>
+                <td v-show="columns.includes('folder')" class="folder"><i class="fad fa-folder"></i> බහාලුම</td>
+                <td v-show="columns.includes('downloads')" class="downloads"><i class="fad fa-tally"></i><span class="ss-hide"> බාගැනීම් ගණන</span></td>
+                <td v-show="columns.includes('date_added')" class="date_added"><i class="fad fa-calendar"></i><span class="ss-hide"> එක් කළ දිනය</span></td>

```

Comment:
බාගත කිරීම් ගණන sounds better. 
Can you show us the source of the word බහාලුම being used for Folder?